app/javascript/elm/src/MobileSessionsFilters.elm

```diff
@@ -92,6 +99,13 @@ update msg model =
         RemoveProfile profile ->
             Labels.removeLabel profile model.profiles (updateProfiles model) Ports.updateProfiles
 
+        UpdateTimeRange value ->
+            let
+                newTimeRange =
+                    TimeRange.update model.timeRange value
+            in
+            ( { model | timeRange = newTimeRange }, Ports.updateTimeRange value )
```

Comment:
I'm not sure I understand the need of the port going back to JS. Doesn't JS already know the value of the range since it was selected there? Maybe we don't need to tell JS back?